spyder/widgets/fileswitcher.py

```diff
@@ -696,9 +704,8 @@ def setup_file_list(self, filter_text, current_path):
                 title = self.widgets[index][1].get_plugin_title().split(' - ')
                 if plugin != title[0]:
                     plugin = title[0]
-                    text += ("<br><big style='color:{0:}'>"
-                             "<b>{1:}</b></big><br>").format(ima.MAIN_FG_COLOR,
-                                                             plugin)
+                    text += ("<span style='color:{0:}'>{1:}"
```

Comment:
Shouldn't this be ``u"..."`` like the rest? Also, add space after ``color:`` for style consistency.